content/python/concepts/built-in-functions/terms/type/type.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'type()'
+Description: 'A function that returns the data type of an object.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Built-in Functions'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+A function that returns the data type of an object.
+
+## Syntax
+
+```pseudo
+type(some_object)
+```
+
+## Example
+
+The `type()` function can be used to confirm the data type of the object `spam`:
```

Comment:
```suggestion
The `type()` function can be used to confirm the data type of the `spam` object:
```

Reply:
same as above

Reply:
@yangc95 made edits and left a couple of comments on your feedback. Should be all set, happy to discuss any points or additional concerns further. Thanks